interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```
@@ -258,27 +270,27 @@ public function getSdohData($pid, $encounter)
     ";
         $row = sqlQuery($sql, ['disability_status', 'pregnancy_status', $pid]) ?: [];
 
-        $disability_code = ($row['disability_code'] ?? '');
-        $disability_status = $row['disability_title'] ?? '';
-        $pregnancy_code = ($row['pregnancy_code'] ?? '');
+        $disability_code = $this->scalarToString($row['disability_code'] ?? '');
+        $disability_status = $this->scalarToString($row['disability_title'] ?? '');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$disability_status'. (undefined)

(UnusedLocalVariable)